litmus-portal/frontend/src/pages/Community/styles.ts

```diff
@@ -32,10 +23,12 @@ const useStyles = makeStyles((theme) => ({
       flexDirection: 'column',
     },
   },
+
   header2: {
     fontSize: theme.spacing(3),
     padding: '1rem 0',
```

Comment:
fontsize in rem and padding should use theme.spacing

Reply:
Changed.